editor/src/DocumentWindows/MaterialInspector.cpp

```
@@ -0,0 +1,102 @@
+
+            if (ImGui::Begin("Material Inspector", &inspectorWindow->getSubInspectorVisibility<MaterialInspector>(), window_flags))
+            {
+            	if (m_firstOpened)
+             		firstDockSetup("Material Inspector");
+                renderMaterialInspector(selectedEntity);
+            }
+            ImGui::End();
+        }
+	}
+
+	void MaterialInspector::update()
```

Cppcheck (2.10-2)

[style] The function 'update' is never used.

(unusedFunction)